### ISSUE TITLE
Support localisation

### ISSUE THREAD

**Issue by bitigchi** [trunkmaster/nextspace#404]
I am not sure if this is accurate, but there is no easy way to localise the "Workspace" application, or not obvious enough.

I see that there are some strings files for some other applications, but not for every one.

**Comment by armm77** [comment]
hi, @bitigchi,  if you mean the location of the workspace, find it on the path /usr/NextSpace/Apps/

**Comment by bitigchi** [comment]
Thanks for the reply. I was asking about the localisation of the desktop environment, like, into different languages. I see `.lproj` files, but most of them seem to lack the string files for the messages.

**Comment by armm77** [comment]
I understand, for the other languages they are not implemented at the moment.
https://github.com/trunkmaster/nextspace/tree/master/Applications/Workspace/English.lproj

**Comment by trunkmaster** [comment]
@bitigchi you're right, there is no localization for Workspace yet. If you want to learn about localization in GNUstep, here is the good starting point - http://wiki.gnustep.org/index.php/Adding_a_new_Natural_Language_to_GNUstep.
Although some work on source code need to be applied.

**Comment by bitigchi** [comment]
Oh, this is interesting! I was planning to translate some strings offline, but in a couple of weeks I'll have the time to set up a CentOS machine for playing with it more comfortably.

I was wondering about what `.gorm` was, now I see. I'll report back with the results.

**Comment by bitigchi** [comment]
Do we need to use the specific versions mentioned in the installation notes, or any latest version is good?

**Comment by trunkmaster** [comment]
Please use recommended versions. They're based on latest stable release of GNUstep libraries with some specific patches.

**Comment by trunkmaster** [comment]
> Interesting... maybe I will contribute as well, can't English .lproj be used as a template ?

Sure
